lib/ReadsUtils/ReadsUtilsImpl.py

```diff
@@ -625,6 +602,56 @@ def validateFASTA(self, ctx, params):
         # return the results
         return [out]
 
+    def calculate_fq_stats(self, reads_object, file_path):
+        eautils = kb_ea_utils(self.callback_url, service_ver='dev')
+        ea_report = eautils.get_ea_utils_stats({'read_library_path': file_path})
+#        print "Full Report : {}".format(ea_report)
+
+        report_lines = ea_report.splitlines()
```

Comment:
Pinged Mike and Bill about making it a task in the future to change EA utils sdk method's behavior.